<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIPanel</class>
 <widget class="QWidget" name="GUIPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUIPanel</string>
  </property>
  <widget class="QTabWidget" name="Tareas">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>661</width>
     <height>361</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="Tarea1">
    <attribute name="title">
     <string>Primera parte</string>
    </attribute>
    <widget class="QSplitter" name="splitter_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>151</width>
       <height>121</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="handleWidth">
      <number>4</number>
     </property>
     <property name="childrenCollapsible">
      <bool>true</bool>
     </property>
     <widget class="QCheckBox" name="rojo">
      <property name="text">
       <string>Rojo</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="verde">
      <property name="text">
       <string>Verde</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="azul">
      <property name="text">
       <string>Azul</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="ModeButton">
      <property name="text">
       <string>Modo RGB</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="Interrupts">
      <property name="text">
       <string>Evento as√≠ncrono</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="checkSWs">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Consultar Pulsadores</string>
     </property>
    </widget>
    <widget class="QwtKnob" name="Knob">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>129</width>
       <height>134</height>
      </rect>
     </property>
     <property name="upperBound">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>0.500000000000000</double>
     </property>
     <property name="singleSteps">
      <UInt>0</UInt>
     </property>
     <property name="pageSteps">
      <UInt>10</UInt>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>6</y>
       <width>221</width>
       <height>27</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="serialPortLabel">
      <property name="text">
       <string>Puerto Serie:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="serialPortComboBox"/>
    </widget>
    <widget class="QPushButton" name="runButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>98</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Inicio</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pingButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>50</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Ping</string>
     </property>
    </widget>
    <widget class="Led" name="LEDsw1">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>200</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Estado Pulsador 1</string>
     </property>
     <property name="whatsThis">
      <string>Led indicator</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="color">
      <color>
       <red>0</red>
       <green>255</green>
       <blue>0</blue>
      </color>
     </property>
    </widget>
    <widget class="Led" name="LEDsw2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>200</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Estado Pulsador 2</string>
     </property>
     <property name="whatsThis">
      <string>Led indicator</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="color_widgets::ColorWheel" name="colorWheel">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>70</y>
       <width>161</width>
       <height>151</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
    <zorder>splitter_2</zorder>
    <zorder>checkSWs</zorder>
    <zorder>Knob</zorder>
    <zorder>splitter</zorder>
    <zorder>runButton</zorder>
    <zorder>pingButton</zorder>
    <zorder>LEDsw1</zorder>
    <zorder>LEDsw2</zorder>
    <zorder>colorWheel</zorder>
    <zorder>pushButton</zorder>
    <zorder>statusLabel</zorder>
    <zorder>pushButton</zorder>
    <zorder>pushButton</zorder>
    <zorder>statusLabel</zorder>
   </widget>
   <widget class="QWidget" name="Tarea2">
    <attribute name="title">
     <string>Segunda Parte</string>
    </attribute>
    <widget class="QPushButton" name="SetTimerOn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Activar Timer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CheckADC">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Leer valor ADC</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="LCDnumber3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="LCDnumber2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="LCDnumber1">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>10</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="LCDnumber0">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>10</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="LCDtext3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu Light</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>PUERTO PE3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="LCDtext2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>50</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu Light</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>PUERTO PE2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="LCDtext1">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>50</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu Light</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>PUERTO PE1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="LCDtext0">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>50</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu Light</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>PUERTO PE0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>511</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Detenido</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Estado</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Led</class>
   <extends>QWidget</extends>
   <header>analogwidgets/led.h</header>
  </customwidget>
  <customwidget>
   <class>color_widgets::ColorWheel</class>
   <extends>QWidget</extends>
   <header>color_wheel.hpp</header>
  </customwidget>
  <customwidget>
   <class>QwtKnob</class>
   <extends>QWidget</extends>
   <header>qwt_knob.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>serialPortComboBox</tabstop>
  <tabstop>pingButton</tabstop>
  <tabstop>runButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
